test/unit/SourceLocator/Type/AbstractSourceLocatorTest.php

```diff
@@ -22,28 +23,28 @@ class AbstractSourceLocatorTest extends TestCase
 {
     public function testLocateIdentifierCallsFindReflection() : void
     {
-        /** @var Reflector|MockObject $mockReflector */
         $mockReflector = $this->createMock(Reflector::class);
+        assert($mockReflector instanceof Reflector || $mockReflector instanceof MockObject);
```

Comment:
Since newer phpunit versions, we can drop these, since direct `$this->createMock()` calls (without builder) can be correctly inferred AFAIK